### DISCUSSION TITLE
大佬，能推出ipq40xx内核5.10或5.15吗？

### DISCUSSION THREAD

**Discussion by cejpo001** [coolsnowwolf/lede#9666] [Ideas / 建议]
大佬，能推出ipq40xx内核5.10或5.15吗？因为现在很多插件新版本多只支持内核5.10以上，比如 Aria2，网上找的在5.4上怎么也跑不起来

**Comment by aiamadeus** [comment]


**Reply by cejpo001** [reply]
> ？都一样的，依赖内核的你只能自己编译进去。

水平有限，试了几回添加内核，报错。。又搜索引擎上搜索的办法也没解决，不知道还要修改哪些地方了
